_overviews/core/string-interpolation.md

```diff
@@ -105,42 +127,106 @@ In Scala, all processed string literals are simple code transformations.   Anyti
     id"string content"
 
 it transforms it into a method call (`id`) on an instance of [StringContext](https://www.scala-lang.org/api/current/scala/StringContext.html).
-This method can also be available on implicit scope.   To define our own string interpolation, we simply need to create an implicit class that adds a new method
-to `StringContext`.  Here's an example:
+This method can also be available on implicit scope.
+To define our own string interpolation, we need to create an implicit class (Scala 2) or an `extension` method (Scala 3) that adds a new method to `StringContext`.
```

Comment:
instead of mentioning both Scala 2 and Scala 3 in this sentence, you could add a specialised line of text to either tab

Reply:
It doesn't really "look well" if I do that, IMO.